frontend/src/routes/FacilityBooking/ViewConflicts/ConflictBookingModal.tsx

```diff
@@ -20,16 +21,18 @@ export default function ConflictBookingModal(props: Props) {
   const toggleStatus = () => {
     props.setModalOpen(!open)
     dispatch(setBookingStatus(BookingStatus.INITIAL))
+    dispatch(resetBooking())
   }
 
-  const onRightClick = () => {
+  const bookNonConflictedBookings = () => {
     // proceed with non-conflict bookings
     dispatch(
       handleCreateNewBooking(
         booking?.facilityID,
         booking?.eventName,
         booking?.startTime ?? null,
         booking?.endTime ?? null,
+        booking?.endDate !== 0,
         booking?.endDate,
         booking?.ccaID,
         booking?.description,
```

Comment:
no context comment:
isit easier to pass the whole booking object and do the truefalse checks in the action?

Reply:
> good to split PRs more, into indiv fixes.
> 
> rationale: if one thing spoils the whole app, can just roll back one indiv PR.

re to this: yes, from the earlier releases, resolving merge conflicts from a chain of 200 commits is  not very fun and has a lot of errors as well